hdr.go

```diff
@@ -292,21 +292,33 @@ func (h *Histogram) Equals(other *Histogram) bool {
 // Export returns a snapshot view of the Histogram. This can be later passed to
 // Import to construct a new Histogram with the same state.
 func (h *Histogram) Export() *Snapshot {
+	var counts []int64
+	found := int64(0)
+
+	for i := int32(0); i < h.countsLen; i++ {
+		if found >= h.totalCount {
+			break
+		}
+		counts[i] = h.counts[i]
```

Comment:
There seem to be at least a couple things wrong here. `counts` is never allocated and `found` is not being incremented.